tests/scripts/multus-test.sh

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env bash
+set -eEuo pipefail
+
+##
+## This script will install a configurable number of multus-networked daemonsets to test that your
+## multus configuration and network hardware can provide enough IP addresses for Ceph.
+## Configuration options are commented below.
+##
+## TODO: in the future, this test should also check network stability under a reasonable amount of
+##       load to/from daemons.
+##
+
+## A comma-delimited list of multus networks to use for test pods
+: "${MULTUS_NETWORKS:="public-net,cluster-net"}"
+
+## The number of daemons to run per node as a test. This should reflect the maximum number of
+## daemons that could run on a single node for your Ceph install. The default value assumes that one
+## daemon type will run on one node, but that there will be 3 OSDs on said node. As an easy
+## guideline, add one for each additional OSD beyond 3 that might run simultaneously on any node.
```

Comment:
Ok so the point is to test that there are enough ip addresses for all the daemons, and perhaps apply some level of load testing in the future. I think just leaving it at that would be sufficient. Or maybe I'm still just too hard-headed to follow the example with OSD count.